src/__tests__/__snapshots__/role-helpers.js.snap

```diff
@@ -0,0 +1,166 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`logRoles logs expected roles for various dom nodes 1`] = `
+"region ####################################
```

Comment:
Rather than `##################` could we use `----------------------`?

Reply:
Switched [deb5bf5c](https://github.com/testing-library/dom-testing-library/pull/285/commits/de5bf5c2a184ed999e80e33b7ec3f05af767b22c)